### PR TITLE
DEVPROD-7272: document region param in s3 commands

### PR DESCRIPTION
**Pull request by dylrich**
DEVPROD-7272

### Description
This parameter was missing from the documentation, but is supported

### Testing
No testing needed

### Documentation
Fixed!

### CODE REVIEW

File: docs/Project-Configuration/Project-Commands.md [hunk 1]
```diff
@@ -1202,6 +1203,7 @@ Parameters:
     `local_file`
 -   `remote_file`: the S3 path to get the file from
 -   `bucket`: the S3 bucket to use.
+-   `region`: AWS region of the bucket, defaults to us-east-1.
```
Comment (Kimchelly) [inline docs/Project-Configuration/Project-Commands.md:1206]: This parameters documentation has to be added to s3.put as well.

**Review by Kimchelly — APPROVED** [review]
LGTM mod one more copy-paste needed.

File: docs/Project-Configuration/Project-Commands.md (hunk 1 above)
Comment (dylrich) [reply docs/Project-Configuration/Project-Commands.md:1206]: S3.put actually already has this documented, I just thought it'd be nice to put it in the example as well

**Review by Kimchelly — APPROVED** [review]
LGTM! Btw, approval usually means there's no need to re-request review unless there's a non-trivial change or concern that would warrant another look.

**Pull request merged**